src/tokenFactory/tokensFactory.ts

```diff
@@ -21,7 +23,8 @@ export const tokensFactory = {
 function createDID(network: string, proxyAddress: string, networkID: string) {
   // network -> "quor" / "fabr"
   // networkID -> redT,...
-  return `did:ala:${network}:${networkID}:${proxyAddress}`
+  const proxyAddressParsed = AddressUtils.getAddressWithoutHexPrefix(proxyAddress)
+  return `did:ala:${network}:${networkID}:${proxyAddressParsed}`
```

Comment:
```suggestion
  return `did:ala:${network}:${networkID}:${proxyAddress}`
```

This functions must allow to create a DID with and without `0x` in the proxy address.

Reply:
Ok, i changed this as sugested and also added this changes to avoid problems:

https://github.com/alastria/alastria-identity-lib/pull/153/files#diff-1f5bce98ae1ea62d450560c15c5f9a9ddcede34f598c4e4b695da3c586785560L25-L27